app/(saas)/manage-projects/[projectSlug]/layout.tsx

```
@@ -31,6 +31,9 @@ import { RewardDetailSidepanel } from "@/shared/panels/reward-detail-sidepanel/r
 import { withAuthenticated } from "@/shared/providers/auth-provider";
 import { PosthogCaptureOnMount } from "@/shared/tracking/posthog/posthog-capture-on-mount/posthog-capture-on-mount";
 import { Translate } from "@/shared/translation/components/translate/translate";
+import { TypographyP } from "@/shared/ui/typography";
```

ESLint

[error] 'TypographyP' is defined but never used.

(unused-imports/no-unused-imports)